NineChronicles.Headless.Executable/Program.cs

```diff
@@ -66,87 +66,102 @@ static void ConfigureSentryOptions(SentryOptions o)
 
         [PrimaryCommand]
         public async Task Run(
-            [Option("app-protocol-version", new[] { 'V' }, Description = "App protocol version token")]
+            [Option("app-protocol-version", new[] { 'V' },
+                Description = "App protocol version token.")]
             string appProtocolVersionToken,
-            [Option('G')]
+            [Option('G',
+                Description = "Genesis block path of blockchain. Blockchain is recognized by its genesis block.")]
             string genesisBlockPath,
-            bool noMiner = false,
-            [Option('H')]
+            [Option('H',
+                Description = "Hostname of this node for another nodes to access. " +
+                              "This is not listening host like 0.0.0.0")]
             string? host = null,
-            [Option('P')]
+            [Option('P',
+                Description = "Port of this node for another nodes to access.")]
             ushort? port = null,
             [Option("swarm-private-key",
                 Description = "The private key used for signing messages and to specify your node. " +
                               "If you leave this null, a randomly generated value will be used.")]
             string? swarmPrivateKeyString = null,
+            [Option("workers", Description = "Number of workers to use in Swarm")]
+            int workers = 5,
+            [Option(Description = "Disable block mining.")]
+            bool noMiner = false,
+            [Option("miner-count", Description = "The number of miner task(thread).")]
+            int minerCount = 1,
             [Option("miner-private-key",
                 Description = "The private key used for mining blocks. " +
                               "Must not be null if you want to turn on mining with libplanet-node.")]
             string? minerPrivateKeyString = null,
             [Option(Description = "The type of storage to store blockchain data. " +
-                                  "If not provided, \"LiteDB\" will be used as default. " + 
+                                  "If not provided, \"LiteDB\" will be used as default. " +
                                   "Available type: [\"rocksdb\", \"memory\"]")]
             string? storeType = null,
+            [Option(Description = "Path of storage. " +
+                                  "This values is required if you use persistent storage e.g. \"rocksdb\"")]
```

Comment:
Because the `storePath` variable is a unary option, not an array option, it should be `value` instead of `values`, doesn't it?

```suggestion
                                  "This value is required if you use persistent storage e.g. \"rocksdb\"")]
```

Reply:
About [the change for this suggestion](https://github.com/planetarium/NineChronicles.Headless/pull/1452/commits/4666c18971d9d3ab0db4bc4b05a86c5ef8d0be9d), the README should also be updated. 😭 

p.s. I hope #1376 is achieved with [planetarium/Cocona.Docs](https://github.com/planetarium/Cocona.Docs) 🙏🏻

Reply:
@moreal OK I'll pull and apply all the changes from Program.cs to README.md
Wait a sec plz.

Reply:
And I'll check about #136 and Cocona.docs :raised_back_of_hand: